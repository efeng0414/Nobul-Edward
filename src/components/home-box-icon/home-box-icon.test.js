import React from "react";
import renderer from "react-test-renderer";
import BoxIcon from "./index";

describe("BoxIcon", () => {
  const componentProps = {
    iconSrc: "",
    hoverIconSrc: "",
    getStarted: true
  };
  const componentJSX = <BoxIcon {...componentProps} />;

  describe("Component Snapshot", () => {
    const componentSnapshot = renderer.create(componentJSX).toJSON();
    it("should match existing snapshot", () => {
      expect(componentSnapshot).toMatchSnapshot();
    });
  });
});
