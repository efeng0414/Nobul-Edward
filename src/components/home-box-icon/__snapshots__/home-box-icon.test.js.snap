// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BoxIcon Component Snapshot should match existing snapshot 1`] = `
<div
  className="box-icon-home"
>
  <img
    className="primary-icon"
    src=""
  />
  <img
    className="hover-icon"
    src=""
  />
  <img
    className="check-icon"
    src={
      Object {
        "default": "file-mock",
      }
    }
  />
</div>
`;
